tests/backup/backup_orphaned_test.go

```diff
@@ -0,0 +1,422 @@
+				log.FailOnError(err, "failed to fetch user %s ctx", user)
+				userOwnerID, err := portworx.GetSubFromCtx(nonAdminCtx)
+				log.FailOnError(err, "failed to fetch user owner id %s", user)
+				backupNamesByOwnerID, err := GetAllBackupNamesByOwnerID(userOwnerID, orgID, ctx)
+				log.FailOnError(err, "failed to fetch backup names with owner id %s from the admin", userOwnerID)
+				for backupName := range userBackupMap[user] {
+					if !IsPresent(backupNamesByOwnerID, backupName) {
+						err := fmt.Errorf("backup %s is not listed in backup names %s", backupName, backupNamesByOwnerID)
+						log.FailOnError(fmt.Errorf(""), err.Error())
+					}
+				}
+			})
+			Step(fmt.Sprintf("Verify backup schedules of the user %s from the admin", user), func() {
+				log.InfoD(fmt.Sprintf("Verifying backup schedules of the user %s from the admin", user))
+				nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+				log.FailOnError(err, "failed to fetch user %s ctx", user)
+				userOwnerID, err := portworx.GetSubFromCtx(nonAdminCtx)
+				log.FailOnError(err, "failed to fetch user owner id %s", user)
+				backupScheduleNamesByOwnerID, err := GetAllBackupScheduleNamesByOwnerID(userOwnerID, orgID, ctx)
+				log.FailOnError(err, "failed to fetch backup schedule names with owner id %s from the admin", userOwnerID)
+				for _, backupScheduleName := range userScheduleNameMap {
+					if !IsPresent(backupScheduleNamesByOwnerID, backupScheduleName) {
+						err := fmt.Errorf("backup schedule %s is not listed in backup schedule names %s", backupScheduleName, backupScheduleNamesByOwnerID)
+						log.FailOnError(fmt.Errorf(""), err.Error())
+					}
+				}
+			})
+			Step(fmt.Sprintf("Verify restores of the user %s from the admin", user), func() {
+				log.InfoD(fmt.Sprintf("Verifying restores of the user %s from the admin", user))
+				nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+				log.FailOnError(err, "failed to fetch user %s ctx", user)
+				userOwnerID, err := portworx.GetSubFromCtx(nonAdminCtx)
+				log.FailOnError(err, "failed to fetch user owner id %s", user)
+				restoreNamesByOwnerID, err := GetAllRestoreNamesByOwnerID(userOwnerID, orgID, ctx)
+				log.FailOnError(err, "failed to fetch restore names with owner id %s from the admin", userOwnerID)
+				for _, restoreName := range userRestoreMap[user] {
+					if !IsPresent(restoreNamesByOwnerID, restoreName) {
+						err := fmt.Errorf("restore %s is not listed in restore names %s", restoreName, restoreNamesByOwnerID)
+						log.FailOnError(fmt.Errorf(""), err.Error())
+					}
+				}
+			})
+		}
+		Step(fmt.Sprintf("Add new user to %s group", adminGroup), func() {
+			log.InfoD(fmt.Sprintf("Adding new user to %s group", adminGroup))
+			for _, user := range createUsers(1) {
+				err := backup.AddGroupToUser(user, adminGroup)
+				log.FailOnError(err, "failed to add user %s to the group %s", user, adminGroup)
+				newAdmin = user
```

Comment:
Can we just print the admin user name .It will help in debugging in case of failure

Reply:
Done.